### PR TITLE
Split math test

### PR DESCRIPTION
**Pull request by pxalcantara**
This PR continues the separation of the Math Unit Test began at #173

### CODE REVIEW

**Comment by pxalcantara** [comment]
Continues the process of reducing the size of the test @chapulina and @j-rivero

File: src/Quaternion_TEST.cc [hunk 1]
```diff
@@ -247,65 +247,97 @@ TEST(QuaternionTest, Integrate)
...
+  EXPECT_EQ(q, math::Quaterniond(0.182574, 0.365148, 0.547723, 0.730297));
+}
+
+/////////////////////////////////////////////////
+TEST(QuaternionTest, Math)
+{
+  math::Quaterniond q(IGN_PI*0.1, IGN_PI*0.5, IGN_PI);
+  EXPECT_TRUE(q == math::Quaterniond(0.110616, -0.698401, 0.110616, 0.698401));
+
+  q.Set(1, 2, 3, 4);
```
Comment (chapulina) [inline src/Quaternion_TEST.cc:336]: Did you mean to remove these tests from here?

Comment (pxalcantara) [reply src/Quaternion_TEST.cc:336]: Yes, I intend to remove this part in the future but this piece is still important for the rest of the Math Test Case so, I'll leave it and remove later during the changes in the rest of the test.

Comment (pxalcantara) [reply src/Quaternion_TEST.cc:331]: @chapulina I think I got your point and remove some duplicated tests.

Comment (pxalcantara) [reply src/Quaternion_TEST.cc:331]: Friendly ping @chapulina @j-rivero :wink:

**Review by chapulina — APPROVED** [review]
Thanks!

**Pull request merged**